universal-application-tool-0.0.1/app/services/program/PathNotInBlockException.java

```diff
@@ -9,6 +9,6 @@
   public PathNotInBlockException(BlockDefinition block, Path path) {
     super(
         String.format(
-            "Block (ID %d) in program (ID ?) does not contain path %s", block.id(), path.path()));
+            "Block (ID %d) in program (ID ?) does not contain path %s", block.id(), path.toString()));
```

Comment:
(blocking): the argument can just be `path`, and the `toString()` will be implicitly called by `String.format`.